week-2/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,7 @@
 */
 
 var codeYourFutureIsGreat = true;
+var mozafarIsCool = true;
```

Comment:
I think you're missing a few here. Does your program give the expected result from the comment below?